index.js

```diff
@@ -1,7 +1,21 @@
-var assign = require("object-assign");
-var stream = require("stream");
-var babel  = require("babel-core");
-var util   = require("util");
+var assign   = require("object-assign");
+var stream   = require("stream");
+var babel    = require("babel-core");
+var util     = require("util");
+
+var canCompile = require("babel-cli/lib/babel/util").isCompilableExtension;
+function arrayify(val, mapFn) {
+  if (!val) return [];
+  if (typeof val === "boolean") return arrayify([val], mapFn);
+  if (typeof val === "string") return arrayify(list(val), mapFn);
```

Comment:
You haven't copied over `list` at all, and this might as well just be `val = val.split(',');`